assets/learning-mode/blocks/lesson-blocks/learning-mode-lesson-actions.block.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sensei-lms/course-theme-lesson-actions",
+  "name": "sensei-lms/learning-mode-lesson-actions",
```

Comment:
Unfortunately block names is also a place where we have to remain backward compatible. If someone has a customized template already using these blocks, it will break. To keep compatibility we'd have to:
- In PHP, register the block with both new and old names
- In the editor, add blocks with the old name that convert them to the new ones in a migration

We can also keep the old name for these for now to not add even more to the scope. I think in the future some of the blocks might grow to be usable in any template, and not be specific to learning mode anyway, so could be worth waiting a bit.

Reply:
Okay Iets go with limiting the scope and keeping the old names. Reverted in 9aee272fb97823eeac17afbbe1b34098a2f996b3